spec/features/admin/members/admin_search_member_spec.rb

```diff
@@ -20,9 +20,9 @@
     scenario "Search with non-matching data" do
       fill_in('Email', with: current_user.email)
       click_button('Search')
-      expect(page).to have_content("No users matched that search")
+      expect(page).to have_content("No users matched your search")
```

Comment:
Google has this message:

>Your search - **{term}** - did not match any documents.
>Suggestions:
>
>Make sure that all words are spelled correctly.
- Try different keywords.
- Try more general keywords.
- Try fewer keywords.

For future reference, how about 
>Your search - **{term}** - did not match any {label.pluralized}.